avro_schema/utils.lua

```diff
@@ -7,6 +7,45 @@ local function table_contains(t, xval)
     return false
 end
 
+-- This function copies fields from one table to the other.
+--
+-- The precise behavior may be controlled with help of `opts` arg.
+-- opts: precise copy settings
+--     fields: table of field keys (names) which should be copied;
+--             if `nil` then all fields copied
+--     except: table of field keys (names) which should not be copied;
+--             works if `fields` are not passed
```

Comment:
Either assert or implement that case, because in the current state it is possible pitfall.